starterkits/primary/src/js/.modernizrrc.js

```diff
@@ -0,0 +1,13 @@
+// Full list of supported options can be found in [config-all.json](https://github.com/Modernizr/Modernizr/blob/master/lib/config-all.json).
```

Comment:
should this file be there? if it is equal with basic?

Reply:
removed